ci-operator/step-registry/openshift/microshift/includes/openshift-microshift-includes-commands.sh

```diff
@@ -527,42 +531,49 @@ function ci_custom_link_report() {
     </thead>
     <tbody>
 EOF
-    
-    for test in "${ARTIFACT_DIR}"/scenario-info/*; do
-        junit_file=""
-        # RF, ginkgo or conformance
-        if [ -f "${test}/log.html" ]; then
-            junit_file="${test}/junit.xml"
-        elif [ -f "${test}/e2e.log" ]; then
-            junit_file="${test}/junit_01.xml"
-        elif [ -f "${test}/ginkgo-results/test-output.log" ]; then
-            junit_file="$( find "${test}/ginkgo-results" -name "junit_e2e_*.xml" -type f | head -1 )"
-        elif [ -f "${test}/gingko-results/test-output.log" ]; then
-            junit_file="$( find "${test}/gingko-results" -name "junit_e2e_*.xml" -type f | head -1 )"
-        fi
 
-        # set scenario status
+    for test in "${ARTIFACT_DIR}"/scenario-info/*; do
         status_class="status-pass"
         status_emoji="✅"
-        if [ -d "${test}/vms/" ]; then
-          if [ ! -f "${junit_file}" ]; then
-            status_class="status-fail"
-            status_emoji="❌"
-          elif [ -f "${junit_file}" ] && grep -q -E '(failures|errors)="[1-9][0-9]*"' "${junit_file}"; then
-            status_class="status-fail"
-            status_emoji="❌"
-          fi
-        elif [ ! -d "${test}/vms/" ] && [ ! -f "${junit_file}" ]; then
-          for temp_junit_file in "$(ls ${test}/phase_*/junit.xml)"; do
-            if grep -q -E 'message="FAILED"' "${temp_junit_file}"; then
+
+        # Determine scenario status
+        ## Check Tests Set Up phase result
+        for phase_junit_file in "$(ls ${test}/phase_*/junit.xml)"; do
+          if grep -q -E 'message="FAILED"' "${phase_junit_file}"; then
```

Comment:
Probably no need for `-E`

Reply:
thanks, addressed in https://github.com/openshift/release/pull/72964/changes/5c1ed218551472c0020d5888524f86e01dab3021 commit